bindings/matrix-sdk-ffi/src/room_list.rs

```diff
@@ -479,6 +516,12 @@ pub struct RoomListItem {
     utd_hook: Option<Arc<UtdHookManager>>,
 }
 
+impl From<matrix_sdk_ui::room_list_service::Room> for RoomListItem {
+    fn from(value: matrix_sdk_ui::room_list_service::Room) -> Self {
+        Self { inner: Arc::new(value), utd_hook: None }
```

Comment:
It's probably wrong to set `utd_hook` to `None`: if the `RoomListItem` is then used to create a timeline with `init_timeline`, then we'll have lost the `utd_hook`. Instead, we need to pass down the `utd_hook` properly from the `RoomListService`.

Reply:
Yup, thanks for the heads up. Done.